<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="plaintextEdit"/>
    </item>
    <item>
     <widget class="QTextEdit" name="keyEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="encryptButton">
      <property name="text">
       <string>加密</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="decryptButton">
      <property name="text">
       <string>解密</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="ciphertextEdit"/>
    </item>
    <item>
     <widget class="QTextEdit" name="decryptedPlaintextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
